platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/metrics/ReconnectMetrics.java

```diff
@@ -90,21 +91,21 @@ public class ReconnectMetrics {
      *
      * @param metrics
      * 		reference to the metrics-system
-     * @param roster the roster reflecting the address book to register metrics for
+     * @param peers the list of peers reflecting the address book to register metrics for
      * @throws IllegalArgumentException if {@code metrics} is {@code null}
      */
-    public ReconnectMetrics(@NonNull final Metrics metrics, @NonNull final Roster roster) {
+    public ReconnectMetrics(@NonNull final Metrics metrics, @NonNull final List<PeerInfo> peers) {
         Objects.requireNonNull(metrics, "metrics");
-        Objects.requireNonNull(roster, "roster");
+        Objects.requireNonNull(peers, "peers");
```

Comment:
I think this is a leftover from old code:
```suggestion
        Objects.requireNonNull(metrics);
        Objects.requireNonNull(peers);
```